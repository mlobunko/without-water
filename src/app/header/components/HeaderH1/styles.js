import styled from 'styled-components';

export const StyledHeaderH1 = styled.h1`
  text-align: center;
  margin-top: 1rem;
  margin-bottom: 3rem;
`;

StyledHeaderH1.displayName = 'StyledHeaderH1';
