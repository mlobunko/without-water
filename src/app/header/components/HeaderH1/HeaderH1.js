import React from 'react';
import PropTypes from 'prop-types';

import { StyledHeaderH1 } from './styles';

export const HeaderH1 = ({ text }) => <StyledHeaderH1>{text}</StyledHeaderH1>;

HeaderH1.propTypes = {
  text: PropTypes.string.isRequired
};

export default HeaderH1;
