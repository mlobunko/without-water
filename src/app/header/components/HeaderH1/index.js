import HeaderH1 from './HeaderH1';

export default HeaderH1;
