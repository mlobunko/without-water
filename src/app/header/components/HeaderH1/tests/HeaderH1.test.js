import React from 'react';
import { shallow } from 'enzyme';

import { HeaderH1 } from '../HeaderH1';

describe('render', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HeaderH1 text={'hola'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
