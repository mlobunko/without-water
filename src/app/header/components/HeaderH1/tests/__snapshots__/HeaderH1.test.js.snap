// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<styled.h1>
  hola
</styled.h1>
`;
